stable/atlantis/README.md

```diff
@@ -1,72 +1,71 @@
+| `aws.config`                                | Contents of a file to be mounted to `~/.aws/config`.                                                                                                                                                                                                                                                      | n/a     |
+| `aws.credentials`                           | Contents of a file to be mounted to `~/.aws/credentials`.                                                                                                                                                                                                                                                 | n/a     |
+| `bitbucket.user`                            | Name of the Atlantis Bitbucket user.                                                                                                                                                                                                                                                                      | n/a     |
+| `bitbucket.token`                           | Personal access token for the Atlantis Bitbucket user.                                                                                                                                                                                                                                                    | n/a     |
+| `bitbucket.secret`                          | Webhook secret for Bitbucket repositories (Bitbucket Server only).                                                                                                                                                                                                                                        | n/a     |
+| `bitbucket.baseURL`                         | Base URL of Bitbucket Server installation.                                                                                                                                                                                                                                                                | n/a     |
+| `environment`                               | Map of environment variables for the container.                                                                                                                                                                                                                                                           | `{}`    |
+| `imagePullSecrets`                          | List of secrets for pulling images from private registries.                                                                                                                                                                                                                                               | `[]`    |
```

Comment:
just a small nit, but the README lists the default as an empty `[]` but it's actually commented out, and therefor `nil` - looking at the functionality in the statefulset I think `nil` is the better default, but maybe then maybe the README should present it that??

Reply:
I'm not sure it makes a difference looking at the API docs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/

Reply:
no, I don't think it makes much difference either

my point was more to do with the readme and the actual implementation are different. but even that doesn't make a difference, because they still render the same - I guess I was just suggesting to keep them the same, but thinking about it more i guess `[]` is the better option in the readme, and then `imagePullSecrets: []` be set in the values 
but again, I don't see it as that important, and if i get approved as a collaborator i'll add a `lgtm` with our without this being changed